common/perception_utils/include/perception_utils/geometry.hpp

```diff
@@ -0,0 +1,60 @@
+// Copyright 2022 Tier IV, Inc.
```

Comment:
```suggestion
// Copyright 2022 TIER IV, Inc.
```
same comment for other files :pray:

Reply:
![image](https://user-images.githubusercontent.com/20228327/180234981-b6eb6325-5240-4647-a9c0-9555e45f44b9.png)
I was planning to deal with it with another PR, but thanks!

Reply:
Give me lgtm!